### PR TITLE
Remove integrated circuits

### PR DESCRIPTION
**Pull request by SirRichardFrancis**
## About The Pull Request

This PR effectively reverts #6416 and #912 (circuits were added twice, so we have that code partially duplicated), removing over 17 thousands lines of pretty much useless code.

## Why It's Good For The Game

A while back I wanted that feature a lot, but now, after over a year since it have been added, I have to admit that it didn't work out. At all.

We have rather outdated version with little content, and most of the stuff that is present - just does not work on our build.
Even very simple constructs like a flashlight is bugged, let alone more complex circuits like chemistry or weapon related stuff.

There is also very few things that are worth making with circuits. Even if Eris were to update and fix everything that is wrong about this feature, completely rooting out exploits and bugs (good luck with that), there is still hardly any reason to have circuits.
They are only encouraging people to commit hours-long autism in far corner of Moebius lab, which is far from intended gameplay.

TL;DR: Feature is broken and requires too much time and effort to "fix". Even then it's not worth the trouble.

## Changelog
:cl:
del: integrated circuits
/:cl:

### CODE REVIEW

**Comment by pexel5** [comment]
nooo, my implants that give me my meth on command, just not that!

**Comment by SirRichardFrancis** [comment]
> nooo, my implants that give me my meth on command, just not that!

Get an organoid.

**Comment by Pink-Chink** [comment]
Heads up, there will be a merge conflict.

**Review by hyperioo — APPROVED** [review]
